homeassistant/components/schlage/strings.json

```diff
@@ -15,5 +15,15 @@
     "abort": {
       "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
     }
+  },
+  "entity": {
+    "switch": {
+      "beeper": {
+        "name": "Keypress Beep"
```

Comment:
is "Keypress Beep" like a product or function name from schlage?

Reply:
Yeah, that's what they call it in their app. I thought that would be more intuitive for users to have it match.

Reply:
👍🏻